fetch.bs

```diff
@@ -4377,8 +4425,14 @@ these steps:
    <li><p>Let <var>serviceWorkerStartTime</var> be the <a for=/>coarsened shared current time</a>
    given <var>fetchParams</var>'s <a for="fetch params">cross-origin isolated capability</a>.
 
+   <li><p>Let <var>onServiceWorkerFetchStarted</var> given <a>service worker fetch controller</a>
+   <var>controller</var>: Set <var>fetchParams</var>'s <a for="fetch params">controller</a>'s
+   <a for="fetch controller">service worker controller</a> to <var>controller</var>. [[!SW]]
+
    <li><p>Set <var>response</var> to the result of invoking <a for=/>handle fetch</a> for
-   <var>requestForServiceWorker</var>. [[!HTML]] [[!SW]]
+   <var>requestForServiceWorker</var> with <var>fetchParams</var>'s
+   <a for="fetch params">crossOriginIsolatedCapability</a>, and
+   <var>onServiceWorkerFetchStarted</var>. [[!SW]]
```

Comment:
Here too I thought the setup was going to be that handle fetch returns a controller that we abort if our controller is aborted. And that handle fetch would run in parallel after that, invoking a callback with a response at some point.

Reply:
Revised based on our conversation back to this simple way of working, with "when aborted" as a side effect inside service workers.